contracts/fees/RevoFees.sol

```diff
@@ -125,12 +179,16 @@ contract RevoFees is Owned, IRevoFees {
         uint256 interestEarnedDenominator
     )
         external
-        pure
+        view
         override
         returns (uint256 feeNumerator, uint256 feeDenominator)
     {
-        // 0.25% (ignores interest earned for simplicity)
-        feeNumerator = 25;
-        feeDenominator = 10000;
+        if (useDynamicWithdrawalFees && interestEarnedDenominator > 0) {
```

Comment:
Why do we check that `interestEarnedDenominator` is >0 here? If no interest was earned, why do we default to the regular fee as a backup?